tests/test_data/config/defaults.yml

```diff
@@ -7,6 +7,16 @@ dry_run: true
 # gitlab api base url without trailing slash
 gitlab_base_url: "https://git.example.com/api/v4"
 
+# Enable tls certification verification. Disable for local development.
+verify: true
+
+# Auto-generate the issue id in Gitlab. If set to `true`, create a Gitlab issue with
+# the same id as the Bugzilla ticket number.  The group members which are 
+# mapped in the user mappings need to be defined as project owners for this
+# to work properly. Otherwise the Gitlab API will ignore the `iid` setting fallback
+# to the default behavior (i.e. auto-generate the issue id).
```

Comment:
```suggestion
# to work properly. Otherwise the Gitlab API will silently ignore the `iid` setting 
# and fallback to the default behavior (i.e. auto-generate the issue id).
```